site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+
+**The extensions mechanism**&emsp;This provides a consistent way for users to install and activate
+extensions.
+
+**Extensions APIs**&emsp;The [extensions APIs](/docs/extensions/reference/) allow the extension's
+code to access features of the browser itself, activating tabs, modifying net requests, and so on.
+
+To create an extension, you assemble some resources -- JavaScript and HTML files, a manifest,
+images, and others -- that constitute the extension. For development and testing, you can load these
+"unpacked" into Chrome using extension developer mode. Once you are happy with your extension, you
+can package it and distribute it to users.
+
+### How do users get extensions?
+
+Most Chrome users get extensions from the Chrome Web Store (CWS). Developers across the globe
+publish their extensions in the Chrome Web Store where they are reviewed and made available to end
+users.
+
+{% Aside %}
+
+Some organizations use enterprise policies to install extensions on their user's devices. These
+extensions may either be fetched from the Chrome Web Store or hosted on the organization's web servers.
+{# TODO: See XXX for info on the capabilities and limitations of self hosting. #}
+
+{% endAside %}
+
+You can distribute your extensions through the [Chrome Developer Dashboard][4], publishing them to the [Chrome
+Web Store][5]. For more information, see the Chrome Web Store [developer documentation][6].
+
+### A note about extensions policy
+
+Extensions on the Chrome Web Store must adhere to the [Chrome Web Store policies][cws-policies].
+Here are some things to keep in mind as you begin:
+
+* An extension must fulfill a [single purpose][1] that is narrowly defined and easy to understand. A
 single extension can include multiple components and a range of functionality, as long as everything
 contributes towards a common purpose.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/bk1NEBu7WXWCXfNJCsms.png",
-       alt="A screenshot of an extension's icon in the browser bar", height="35", width="91" %}
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/XniXB3snAeMvLwI1am3O.png", alt="Screenshot of AMP validator extension pinned", width="169", height="62" %}
 
-User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
-Mail Checker extension][2] shown on the right, to [overriding][3] an entire page.
+* User interfaces should be minimal and have intent. They can range from a simple icon, such as the
+  [AMP validator] extension shown above, to opening a new window with a form, like the [PackageTrack]
+  extension shown below.
```

Comment:
Can we use a google-published extension, rather than PackageTrack?

Reply:
+1. If we do want to use 3rd party extensions we should probably discuss offline what steps are required to do so.

Reply:
@dotproto Any particular recommendation?

Reply:
Replaced with Google Similar Pages. I don't like the "preview unavailable" images, so I'm still shopping...